hack/windows/Prepare-Node.ps1

```diff
@@ -45,9 +38,16 @@ function DownloadFile($destination, $source) {
     }
 }
 
-If (Get-Service kubelet -ErrorAction SilentlyContinue) {
-    Write-Host("Found existing kubelet service, exit.")
-    exit 0
+if ($ContainerRuntime -eq "Docker") {
+    if (-not(Test-Path "//./pipe/docker_engine")) {
+        Write-Error "Docker service was not detected - please install start Docker before calling PrepareNode.ps1 with -ContainerRuntime Docker"
+        exit 1
+    }
+} elseif ($ContainerRuntime -eq "containerD") {
+    if (-not(Test-Path "//./pipe/containerd-containerd")) {
+        Write-Error "ContainerD service was not detected - please install and start containerD before calling PrepareNode.ps1 with -ContainerRuntime containerD"
```

Comment:
dotto, "ContainerD" is not a well-known writing, we removed all such writings once, don't add new.